### ISSUE TITLE
drupalvm.dev can't be reached on Windows 10

### ISSUE THREAD

**Issue by Kachilero** [geerlingguy/drupal-vm#548]
First off let me say Thank You! This is the first VM setup for Drupal that I've found that doesn't just poop itself and sit there crying when ran on Windows.

However after getting everything up and running I am not able to access the site through the host machines browser. Perhaps I missed something or perhaps something went wrong. I don't suppose you'd have any insight into this?

Here is the rundown of what I did:
The initial `vagrant up` error'd out on an `ssh` issue, but after doing as suggested in the "Authentication failure on vagrant up" section of the docs, it continued without an issue

I have both `vagrant-hostupdater` and `vagrant-hostmanager` installed and they seemed to have correctly edited the Windows `hosts` file.

I changed the the synced folder type to `smb` in case this was just an issue of the VM being slow. I then destroyed the box and did `vagrant up` again, but noticed that the `hosts` file was not properly edited. Either way, `drupalvm.dev` was unavailable.

Any help would be appreciated.

**Comment by oxyc** [comment]
What exact message do you get in the browser? From my experience I failed provision sometimes need you to go into the VM and manually restart apache/php5-fpm/mysql.

Try with SSHing into the VM and run:
```
sudo service apache2 restart
sudo service php5-fpm restart
sudo service mysql restart
```

**Comment by Kachilero** [comment]
All I get is a message that says:
```
This site can't be reached
drupalvm.dev took too long to respond
ERR_CONNECTION_TIMED_OUT
```

The `network` tab in Chrome inspector just say `(failed)`

I reverted the folder structure back to `nfs` and did as you suggested, but, alas, the error is still happening.

**Comment by thom8** [comment]
@Kachilero it could be that the domain is not resolving - `ping drupalvm.dev` to see if it resolves to the IP of the VM.

Are you using the hostsupdater plugin? or editing your hosts file directly?

**Comment by Kachilero** [comment]
@thom8 You may have been correct. 
**tl;dr** Removing `vagrant` plugins seems to have worked

I removed `vagrant-hostupdater` and `vagrant-hostsmanager` last night (which, by the way, I was running from `mingw64` with `administrative` privileges,) then left the VM running when I went to bed. This morning I set `build_make` & `install_site` variables in `config.yml` file to false then `provisioned` the box again, then `ping`ed it...and it worked. 

So one of the 3 things I did different this time seemed to have worked. I realize it was bad debugging on my part to not do them one at a time, but I'm leaving this here in case any one else runs into this issue:

* Remove `vagrant-hostupdater` && `vagrant-hostsmanager` plugins
* If site codebase and database have already been installed, set `build_make` && `install_site` to false in `config.yml`
* Don't `vagrant destroy` the box; let it run for a while (?)
* Provision the box again, perhaps several times

Thank you all for the help.

**Comment by geerlingguy** [comment]
@Kachilero - haha, it's like hitting a flashlight a few times if it's not working, or running `drush cc all` or `drush cr` a few times. Just provision and reload a few more times, and things start working :)
